.github/workflows/go.yml

```diff
@@ -55,3 +55,20 @@ jobs:
           SAMPLE_CONTAINER_ID=$(docker run -d -p 8000:8000 --rm -e CLIENT_ID -e CLIENT_SECRET $DOCKER_REGISTRY/$OWNER/oidc-samples/go:latest)
           docker run --network="host" --rm -e CLIENT_ID -e CLIENT_SECRET $DOCKER_REGISTRY/$OWNER/oidc-samples/integration-tests:latest
           docker container stop $SAMPLE_CONTAINER_ID
+
+  run-no-mitm-integration-tests:
```

Comment:
Shouldn't the name be something like `run-behind-proxy`?

Reply:
We'll need to (at some point) run another job, that runs integration tests that does apply tampering with requests through MITM.
My idea was to be called `run-mitm-integration-tests`, because essentially that's what these jobs do. They run integration tests. `run-behind-proxy` doesn't explain what is being run.

Reply:
Maybe `run-proxy-integration-tests`? I just don't like the negative in the name, as it only explains what it's not and not what it is.

Reply:
Doesn't look resolved.